plugins/healthcheck/main.go

```diff
@@ -0,0 +1,110 @@
+package example
+
+import (
+	"flag"	
+	"time"
+	"encoding/json"
+	"net/http"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/node"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/rpc"
+
+	"github.com/openrelayxyz/xplugeth"
+	"github.com/openrelayxyz/xplugeth/hooks/initialize"
+	"github.com/openrelayxyz/xplugeth/types"
+)
+
+var (
+	flags = *flag.NewFlagSet("healthcheck-plugin-flags", flag.ContinueOnError)
+	hcTolerance = flags.Int("healthcheck.tolerance", 36, "tolerance for healthcheck in seconds, default is 36")
+	hcPort = flags.String("healthcheck.port", "9999", "health check port, default is 9999")
+)
+
+
+type healthCheckModule struct {
+	client	*rpc.Client
+	tolerance int
+	port string
+}
+
+func init() {
+	xplugeth.RegisterFlags(flags)
+	xplugeth.RegisterModule[healthCheckModule]("healthcheck")
+}
+
+func (h *healthCheckModule) InitializeNode(s *node.Node, b types.Backend) {
+	h.client = s.Attach()
+
+	h.tolerance = *hcTolerance
+	if h.tolerance == 36 {
```

Comment:
This looks like debugging that should probably be taken out

Reply:
This is a flag check, the way the plugin in written at the moment the port and tolerance (number of seconds old the head block can be for us to report the node as healthy), can be modified away from default values with flags.

Reply:
Then wouldn't you want to log if it's not the default value, rather than if it is the default value?

Reply:
Ah...